### PR TITLE
[ADD] elico_mail_batch_read

### PR DESCRIPTION
**Pull request by noahzaozao**
@seb-elico @elicoidal Please review this PR , it's from superman server and it's missing in the elico_odoo repo.

### CODE REVIEW

File: elico_mail_batch_read/__init__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+##############################################################################
```
Comment (seb-elico) [inline elico_mail_batch_read/__init__.py]: Short headers

File: elico_mail_batch_read/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    In the same view as Inbox, add a "list" view button. When switching to list
+    view, display the following info:
+    - Date: date
+    - From: partner_ids
+    - Subject: subject
+    - Body (only the begginning)
+    - When the user checks one or several emails, add an option in the drop down
+    menu to mark them all as read.
+    ''',
+    'author': 'Elico corp',
```
Comment (seb-elico) [inline elico_mail_batch_read/__openerp__.py]: s/corp/Corp/

File: elico_mail_batch_read/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': 'Mail Batch Read',
```
Comment (seb-elico) [inline elico_mail_batch_read/__openerp__.py:5]: This name is fine. Let's rename the module to `mail_batch_read`

**Comment by elicoidal** [comment]
LGTM: I dont even understand why it is not installed in our ERP...

**Comment by noahzaozao** [comment]
@elicoidal @seb-elico Please review this PR

**Pull request merged**